tests/pytests/integration/cli/test_syndic_eauth.py

```diff
@@ -54,6 +54,9 @@ def syndic_network():
     try:
         network = client.networks.create(name="syndic_test_net", ipam=ipam_config)
         yield network.name
+    except Exception as e:  # pylint: disable=broad-except
```

Comment:
This seems to be un-related to your change, correct? Is this a failure you are seeing locally or in CI/CD? My concern here is if this is something seen in CI/CD it is something that needs to be addressed otherwise this tests will never run.

Reply:
Yes, we see these docker tests failing in some OSes, where docker might not even be available.

I just followed here same approach that was done just few lines before: https://github.com/saltstack/salt/pull/66647/files#diff-e238394d09fe27028aa921893ce67e3f36ab548bf8b4997423beec357016a1b6R45